src/Featurizers/UnitTests/CMakeLists.txt

```diff
@@ -52,6 +53,7 @@ foreach(_test_name IN ITEMS
 
     target_link_libraries(${_test_name} PRIVATE
         FeaturizersCode
+        eigen
```

Comment:
> eigen [](start = 7, length = 6)

You don't need to link to INTERFACE libraries.

Reply:
looks the compiler is not happy without that(can't find Eigen/Dense)